### ISSUE TITLE
DCEG - New style for green palette for external link icon 

### ISSUE THREAD

**Issue by claireblaustein** [NCIOCPL/cgov-digital-platform#2248]
the external feature card on Fellowships gets a external link icon that’s purple  - my guess is this theme needs it’s own. 
Creative team will look at all microsites to provide design requirements for this: https://tracker.nci.nih.gov/browse/CGOV-9968. Work on this ticket should not begin until the JIRA ticket is completed.

https://dceg-prod-acsf.cancer.gov/fellowship-training

![image](https://user-images.githubusercontent.com/39384914/61641262-d286b600-ac6c-11e9-8c41-64c2489077e1.png)

**Comment by claireblaustein** [comment]
Griffy provided a file for this as an SVG in the jira ticket: https://tracker.nci.nih.gov/browse/CGOV-9968

**Comment by bennettcc** [comment]
MVP1 launched yesterday - ticket no longer needed
